aws/csi-driver/main.tf

```diff
@@ -128,18 +127,18 @@ resource "helm_release" "kubernetes_efs_csi_driver" {
   }
 }
 
-resource "kubectl_manifest" "storage_class" {
-  yaml_body  = <<YAML
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: env0-state-sc
-provisioner: efs.csi.aws.com
-reclaimPolicy: ${var.reclaim_policy}
-parameters:
-  provisioningMode: efs-ap
-  fileSystemId: ${var.efs_id}
-  directoryPerms: "700"
-YAML
+resource "kubernetes_storage_class" "storage_class" {
   depends_on = [helm_release.kubernetes_efs_csi_driver]
+
+  storage_provisioner = "efs.csi.aws.com"
+  reclaim_policy = var.reclaim_policy
+  allow_volume_expansion = true
+  metadata {
+    name = "env0-state-sc"
```

Comment:
add is as a parameter - because we support in our chart this parameter

Reply:
@HeverFarber you mean have the ability to have dynamic storage class name?

Reply:
Anyways, OoS ATM. Should be done in a separate PR

Reply:
If a customer uses this chart to create the storage class they won't need to change it in our chart, there is no reason they should both be dynamic.